src/Mod/Material/App/MaterialFilterOptionsPyImp.cpp

```diff
@@ -0,0 +1,122 @@
+/***************************************************************************
+ *   Copyright (c) 2023-2024 David Carter <dcarter@david.carter.ca>        *
+ *                                                                         *
+ *   This file is part of FreeCAD.                                         *
+ *                                                                         *
+ *   FreeCAD is free software: you can redistribute it and/or modify it    *
+ *   under the terms of the GNU Lesser General Public License as           *
+ *   published by the Free Software Foundation, either version 2.1 of the  *
+ *   License, or (at your option) any later version.                       *
+ *                                                                         *
+ *   FreeCAD is distributed in the hope that it will be useful, but        *
+ *   WITHOUT ANY WARRANTY; without even the implied warranty of            *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU      *
+ *   Lesser General Public License for more details.                       *
+ *                                                                         *
+ *   You should have received a copy of the GNU Lesser General Public      *
+ *   License along with FreeCAD. If not, see                               *
+ *   <https://www.gnu.org/licenses/>.                                      *
+ *                                                                         *
+ **************************************************************************/
+
+#include "PreCompiled.h"
+
+#include <QMetaType>
+
+#include <Base/Quantity.h>
+#include <Base/QuantityPy.h>
+#include <CXX/Objects.hxx>
+#include <Gui/MetaTypes.h>
+
+#include "MaterialFilter.h"
+
+#include "MaterialFilterOptionsPy.h"
+
+#include "MaterialFilterOptionsPy.cpp"
```

Comment:
Do you mean to be including this *.cpp file? (I'm trying to isolate why the linter is losing its mind, and infinite inclusion loops come to mind).